src/staticdata.c

```diff
@@ -1096,6 +1331,22 @@ static void jl_write_values(jl_serializer_state *s)
                 jl_code_instance_t *m = (jl_code_instance_t*)v;
                 jl_code_instance_t *newm = (jl_code_instance_t*)&s->s->buf[reloc_offset];
 
+                if (s->incremental) {
+                    arraylist_push(&s->fixup_list, (void*)reloc_offset);
+                    if (m->min_world > 1)
+                        newm->min_world = ~(size_t)0;     // checks that we reprocess this upon deserialization
+                    if (m->max_world != ~(size_t)0)
+                        newm->max_world = 0;
+                    else {
+                        if (m->inferred && ptrhash_has(&s->callers_with_edges, m->def))
+                            newm->max_world = 1;  // sentinel value indicating this will need validation
+                        if (m->min_world > 0 && m->inferred) {
+                            // TODO: also check if this object is part of the codeinst cache
+                            // will check on deserialize if this cache entry is still valid
```

Comment:
@vtjnash I think you left this comment, is this something we need to address here?